AlphaWallet/UI/Form/MnemonicLengthRule.swift

```diff
@@ -15,8 +16,12 @@ struct MnemonicLengthRule<T: Equatable>: RuleType {
     public func isValid(value: T?) -> ValidationError? {
         if let str = value as? String {
             let words = str.trimmed.split(separator: " ")
-            return (words.count != HDWallet.mnemonicWordCount) ? validationError : nil
+            return !isValidSeedPhraseLength(count: words.count) ? validationError : nil
         }
         return value != nil ? nil : validationError
     }
+
+    private func isValidSeedPhraseLength(count: Int) -> Bool {
+        return Features.is24SeedWordPhraseAllowed ? count == 12 || count == 24 : count == 12
```

Comment:
We can use `HDWallet.SeedPhraseCount.word12.count` instead of 12/24 and introduce  a `HDWallet.SeedPhraseCount.count` property. Helps to discover this line if we ever add support for other seed phrase of other lengths.

Reply:
Can we just add an extension function to `HDWallet.SeedPhrase.Count` to return an array of valid lengths? Then just compare the value to the values in the array?

Reply:
static func? Sure. The one I suggested makes it impossible to miss a change and your version keeps it localized. Pros and cons to each. But both works since the type is small.